### PR TITLE
Added download tooltip - solves #221

### PR DESCRIPTION
**Pull request by hhenkel**
This should solve #221.

### CODE REVIEW

**Comment by Bwko** [comment]
LGTM

**Comment by thibaultmeyer** [comment]
LGTM

File: templates/repo/home.tmpl [hunk 1]
```diff
@@ -69,7 +69,7 @@
 						<button class="ui basic icon button poping up clipboard" id="clipboard-btn" data-original="{{.i18n.Tr "repo.copy_link"}}" data-success="{{.i18n.Tr "repo.copy_link_success"}}" data-error="{{.i18n.Tr "repo.copy_link_error"}}" data-content="{{.i18n.Tr "repo.copy_link"}}" data-variation="inverted tiny" data-clipboard-target="#repo-clone-url">
 							<i class="octicon octicon-clippy"></i>
 						</button>
-						<div class="ui basic jump dropdown icon button">
+                        <div class="ui basic jump dropdown icon button poping up" data-content="{{.i18n.Tr "repo.download_archive"}}" data-variation="tiny inverted">
```
Comment (tboerger) [inline templates/repo/home.tmpl]: Nitpick: wrong indentation

Comment (hhenkel) [reply templates/repo/home.tmpl]: Yeah, that's because I'm used to use whitespaces not tabs...will fix it.

**Comment by tboerger** [comment]
Perfect, thanks!

**Comment by tboerger** [comment]
It's missing the bindata update :(

**Pull request merged**